compression_test.go

```diff
@@ -10,7 +10,13 @@ import (
 	"github.com/golang/mock/gomock"
 )
 
-func TestCompression(t *testing.T) {
+const (
+	helloworld               = `hello world!`
```

Comment:
Nit: Use camel case. 
```suggestion
	helloWorld               = `hello world!`
```

Reply:
haha.  Ok FINE.  :)